src/chrome/content/rules/LeanCrew.com.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+	Invalid certificate:
+		- www
+		- admin
+		- eww
+		- mail
+		- vvv
+		- vwww
+		- wew
+		- wqw
+		- wvw
+		- ww2
```

Comment:
This domain uses a wildcard DNS record, so you can remove this comments and just describing this behavior.